pkg/controller/servicebinding/service_test.go

```diff
@@ -215,7 +215,7 @@ func TestBuildOwnerResourceContext(t *testing.T) {
 
 	testCases := []testCase{
 		{
-			inputPath:       "data.user",
+			inputPath:       "data.username",
```

Comment:
It isn't obvious to me what made this change required, could you elaborate?

Reply:
This was made for testing, reverting back